### DISCUSSION TITLE
npm init @eslint/config shows only 2 options for using ESlint

### DISCUSSION THREAD

**Discussion by KV-Vel** [eslint/eslint#18708] [Q&A] [answered]
Hello.

Recently I was trying to use eslint but when i run npm init @eslint/config 
It only gives 2 options, however almost every guide that I watched earlier shows that there should be 3 options available.

node version: v20.11.0
npm version : v10.7.0

![image](https://github.com/user-attachments/assets/fb6685eb-a1ab-43a2-ad68-f24517b6e603)

**Comment by harish-sethuraman** [comment, marked as answer]
Yes the third option to choose style guide was removed as part of https://github.com/eslint/create-config/pull/108. You can still continue to use `npm init @eslint/config@latest -- --config eslint-config-standard` as mentioned in https://eslint.org/docs/latest/use/getting-started#quick-start

**Reply by KV-Vel** [reply]
Thank you for help!
